envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/resource/endpoints/EnvoyEndpointsFactory.kt

```diff
@@ -91,7 +90,10 @@ class EnvoyEndpointsFactory(
                         .addAllEndpoints(assignment.endpointsList?.filter { e ->
                             e.locality.zone == properties.loadBalancing.trafficSplitting.zoneName
                         })
-                        .setClusterName(getSecondaryClusterName(routeSpec.clusterName))
+                        .setClusterName(
+                            "${routeSpec.clusterName}-" + properties.loadBalancing
```

Comment:
it would be better to hide this under the function(like it was before), it is used few times - getSecondaryClusterName this can have second param - config

Reply:
done